src/components/mainPages/SidebarPage.vue

```diff
@@ -58,6 +66,19 @@ export default {
     sidebarModes() {
       return sidebarModesForViews[this.currentSidebarView]?.sidebarModes;
     },
+    sidebarStyles() {
+      const styles = {};
+
+      styles.width = `${this.dividerPosition}px`;
+      styles.transform = this.isSidebarOpen
+        ? 'translateX(0)'
+        : `translateX(${this.dividerPosition}px)`;
+
+      return styles;
+    },
+    isSidebarBreakpointReached() {
+      return this.dividerPosition > +sidebarWidthBreakpoint;
```

Comment:
I think that it's better or export a number (if it's possible) or before component script transorm it into a number const and use it after without "+"